lib/api.ts

```diff
@@ -98,6 +98,25 @@ export async function getContributorBySlug(file: string, detail = false) {
   }
 
   const slackMessages = await getSlackMessages(data.slack);
+  const UtctoDate = (date: number | string) => {
```

Comment:
1. Since this function basically parses Slack / GitHub activity time to Date object, this is a very commonly required feature across the platform. Move this to a utility function.
2. Follow JS naming conventions https://google.github.io/styleguide/jsguide.html#naming-method-names

```suggestion
  const parseActivityTime = (date: number | string) => {
```